src/app/layout/navbar/navbar.component.scss

```diff
@@ -1,3 +1,37 @@
+#app-menu {
+  color:#5F6060;
+  background-color: white;
+  height: 100px;
+}
```

Comment:
There are classes in Materializecss to define color instead of create you own styles.

Reply:
You are right, but in this case those codes were defined by the POs and unfortunately they don't exist on those classes from Materialize